docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -98,12 +98,15 @@ In order to make sure this works for either local or remote clients, we have to
 Docker network and expose the required ports correctly.
 
 There are two ways to launch Alluxio Docker containers on the Docker host:
- + A. Use host network
- + B. Use user-defined bridge network
+ + Use [host network](https://docs.docker.com/network/host/) to
```

Comment:
are you trying to convey that there are 2 options and the user should pick one of them? i would keep the A and B denotation, explain underneath the options that the user should pick 1, then rename by the subtitles to `Option A` and `Option B`. i feel the existing layout is more reasonable since it defines that there are two options, but now they squashed together in this section

Reply:
SG. Will do. The only reason to remove A and B is because the final html output looks confusing. Will expend it to Option A and option B